build_daemon/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 4.0.2-wip
```

Comment:
We could go ahead and publish right away, so I would drop `-wip` from the versions

Reply:
I was imaging adding this code to `package:web_socket_channel`, doing a release and then allowing `build_daemon` to support that version.

Is that too slow?

Reply:
That should be fine, but we could just wait to land this at all then until that is ready?

Reply:
I don't have a strong opinion. So how about I make the changes that you suggested and land this PR. That will take some of the pressure off to land the change in `web_socket_channel` and publish it.

Reply:
SGTM